site/content/docs/manifest/backend-service.en.md

```diff
@@ -1,63 +1,142 @@
 List of all available properties for a `'Backend Service'` manifest. To learn about Copilot services, see the [Services](../concepts/services.en.md) concept page.
 
-???+ note "Sample manifest for an api service"
-
-    ```yaml
-        # Your service name will be used in naming your resources like log groups, ECS services, etc.
+???+ note "Sample backend service manifests"
+
+    === "Service Discovery"
+
+        ```yaml
+            # Your service is reachable at "http://api.${COPILOT_SERVICE_DISCOVERY_ENDPOINT}:8080" only within your VPC.
+            name: api
+            type: Backend Service
+
+            image:
+              build: ./api/Dockerfile
+              port: 8080
+              healthcheck:
+                command: ["CMD-SHELL", "curl -f http://localhost:8080 || exit 1"]
```

Comment:
do we actually need `|| exit 1`? doesn't curl already fail on any non-2xx with `-f`?

Reply:
🤷 https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html it was just copy/pasted from the ECS docs